libindy/src/utils/environment.rs

```diff
@@ -1,13 +1,25 @@
 use std::env;
 use std::path::PathBuf;
+use std::fs;
 
 pub struct EnvironmentUtils {}
 
 impl EnvironmentUtils {
     pub fn indy_home_path() -> PathBuf {
         // TODO: FIXME: Provide better handling for the unknown home path case!!!
+
```

Comment:
This code doesn't look correct for me.

- I see 2 unwraps. Will we crash if there is no EXTERNAL_STORAGE variable?
- Why we create directories here? It isn't responsibility of this function
- Flow also looks very non-obvious? Can you just use on match or if/if else/if else?

Reply:
Libindy will now panic! if it cannot find the EXTERNAL_STORAGE .

Reply:
Agree with @vimmerru. Please fix all 3 points